scrapy/extensions/feedexport.py

```diff
@@ -331,12 +394,13 @@ def _get_exporter(self, file, format, *args, **kwargs):
     def _get_storage(self, uri):
         return self._get_instance(self.storages[urlparse(uri).scheme], uri)
 
-    def _get_uri_params(self, spider, uri_params):
+    def _get_uri_params(self, spider, uri_params, slot):
         params = {}
         for k in dir(spider):
             params[k] = getattr(spider, k)
-        ts = datetime.utcnow().replace(microsecond=0).isoformat().replace(':', '-')
-        params['time'] = ts
+        params['time'] = datetime.utcnow().replace(microsecond=0).isoformat().replace(':', '-')
+        params['batch_time'] = datetime.utcnow().isoformat().replace(':', '-')
```

Comment:
What do you think about calling  datetime.utcnow() once? Otherwise the results may be different.

Reply:
I guess it is very unlikely but there is no need to test our fortune =).
Thank you.